src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -280,6 +285,16 @@ internal Silo(SiloInitializationParameters initializationParams)
             services.AddSingleton<DefaultPlacementStrategy>();
             services.AddSingleton<ClientObserversPlacementDirector>();
 
+            // Versions
+            services.AddSingleton<VersionSelectorManager>();
```

Comment:
It does work, so I suppose it's fine. Did the behaviour of DI construction change in `Microsoft.Extensions.DependencyInjection` v1.1.0?